tests/0004-conf.c

```diff
@@ -381,6 +381,25 @@ int main_0004_conf (int argc, char **argv) {
 		rd_kafka_conf_destroy(conf);
 	}
 
+	{
```

Comment:
I know the source is messy with tab and whitespace indentation, but all new code, and changed lines, should use 8 whitespaces for tab rather than the tab character.

Reply:
It did have me thinking to do a reformat to be honest ... but I couldn't actually work out what the current settings were. Maybe providing an [`.editorconfig` file](http://editorconfig.org/) could help?

Reply:
It is mentioned in the contribution guidelines:
https://github.com/edenhill/librdkafka/blob/master/CONTRIBUTING.md#indenting

The plan is to add something like [astyle](http://astyle.sourceforge.net/astyle.html) to enforce the coding guidelines.